release/release-notes/data-prepper.release-notes-2.0.0.md

```diff
@@ -0,0 +1,53 @@
+* Support Core Peer Forwarder which replaces the peer forwarder processor plugin ([#700](https://github.com/opensearch-project/data-prepper/issues/700))
+* Support parsing CSV and TSV values in event fields using CSV processor ([#1081](https://github.com/opensearch-project/data-prepper/issues/1081))
+* Support processing JSON values in event fields using JSON processor ([#831](https://github.com/opensearch-project/data-prepper/issues/831))
+* Support multi node aggregation using Core Peer Forwarder ([#978](https://github.com/opensearch-project/data-prepper/issues/978))
+* Support peer forwarding in service_map_stateful processor using Core Peer Forwarder ([#1765](https://github.com/opensearch-project/data-prepper/issues/1765))
+* Support peer forwarding in otel_trace_raw processor using Core Peer Forwarder ([#1766](https://github.com/opensearch-project/data-prepper/issues/1766))
+
+
+### Enhancements
+* Support concurrency in Data Prepper expression ([#1189](https://github.com/opensearch-project/data-prepper/issues/1189))
+* Support bulk create option in OpnSearch sink ([#1561](https://github.com/opensearch-project/data-prepper/pull/1561))
+* Added new metric to track utilization rate of buffer ([#1817](https://github.com/opensearch-project/data-prepper/issues/1817))
+* Support publishing metrics for Data Prepper core ([#1789](https://github.com/opensearch-project/data-prepper/issues/1789))
+* Support configurable timeouts for processors and sinks to flush the ata downstream before shutdown ([#1742](https://github.com/opensearch-project/data-prepper/issues/1742))
+* Support for consistent decoding of s3 events ([#1687](https://github.com/opensearch-project/data-prepper/issues/1687))
+* Support Duration in Data Prepper server configuration ([#1623](https://github.com/opensearch-project/data-prepper/issues/1623))
+* Enabled HTTP health check for OTel trace source and OTel metrics source ([#1546](https://github.com/opensearch-project/data-prepper/issues/1546))
+* Made Java 11 to be the baseline version for Data Prepper ([#1422](https://github.com/opensearch-project/data-prepper/issues/1422))
+* Updated HTTP source request timeout to use configured timeout ([#975](https://github.com/opensearch-project/data-prepper/issues/975))
+* Distribute Data Prepper Docker image with JDK 17 ([#694](https://github.com/opensearch-project/data-prepper/issues/694))
+* Support ACM and S3 for TLS/SSL in HTTP source ([#365](https://github.com/opensearch-project/data-prepper/issues/365))
+* Updated HTTP methods supported on core API endpoints ([#313](https://github.com/opensearch-project/data-prepper/issues/313))
+
+### Bug Fixes
+* Fix a bug where file sink fails to write to output file with multiple pipeline threads ([#1843](https://github.com/opensearch-project/data-prepper/issues/1843))
+* Fixed health check bug when Auth is enabled for HTTP and OTel trace source ([#1600](https://github.com/opensearch-project/data-prepper/issues/1600))
```

Comment:
I don't think this is was a bug. I'd consider this an enhancement to allow for an unauthenticated health-check endpoint.